src/xml_parsing.cpp

```diff
@@ -536,6 +536,25 @@ void VerifyXML(const std::string& xml_text,
       }
       else if(search->second == NodeType::CONTROL)
       {
+        if (name == "Sequence" && children_count <= 2)
+        {
+          ThrowError(line_number,
+                     std::string("A Sequence node must have 2 or more children"));
```

Comment:
There's no point to a Sequence with just 1 child

Reply:
nevermind, this is useful for the first Control node in the tree, I guess. Will revert.